packages/material-ui/src/Modal/ModalManager.ts

```diff
@@ -122,16 +134,28 @@ function handleContainer(containerInfo, props) {
   return restore;
 }
 
-function getHiddenSiblings(container) {
-  const hiddenSiblings = [];
-  [].forEach.call(container.children, (node) => {
-    if (node.getAttribute && node.getAttribute('aria-hidden') === 'true') {
```

Comment:
[`container.children` will containe the child elements](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/children). Objects implementing this interface MUST implement `getAttribute`. This check is redundant for "correct" environments. Or is this a browser quirk?